src/transaction/utils.ts

```diff
@@ -51,11 +51,32 @@ function prepareTransaction(txJSON: TransactionJSON, api: RippleAPI,
 ): Promise<Prepare> {
   common.validate.instructions(instructions)
   common.validate.tx_json(txJSON)
+  const disallowedFieldsInTxJSON = ['maxLedgerVersion', 'maxLedgerVersionOffset', 'fee', 'sequence']
+  const badFields = disallowedFieldsInTxJSON.filter(field => txJSON[field])
+  if (badFields.length) {
+    return Promise.reject(new ValidationError('txJSON additionalProperty "' + badFields[0] +
+      '" exists in instance when not allowed'))
+  }
 
   const account = txJSON.Account
   setCanonicalFlag(txJSON)
 
-  function prepareMaxLedgerVersion(): Promise<object> {
+  function prepareMaxLedgerVersion(): Promise<TransactionJSON> {
+    // Up to one of the following is allowed:
+    //   txJSON.LastLedgerSequence
+    //   instructions.maxLedgerVersion
+    //   instructions.maxLedgerVersionOffset
+    if (txJSON.LastLedgerSequence && instructions.maxLedgerVersion) {
+      return Promise.reject(new ValidationError('`LastLedgerSequence` in txJSON and `maxLedgerVersion`' +
+        ' in Instructions cannot both be set'))
```

Comment:
nit: I suggest writing \`instructions` rather than Instructions, to be consistent with how the other parts are written